### PR TITLE
Cleanup RFCs for mkdocs website generation

### PR DESCRIPTION
**Pull request by swcurran**
The vast majority of the changes are just to add "mailto:" to the front of the email addresses in the RFCs -- that was missed in the templates, and it carried into the RFCs themselves.

A few other cleanups of paths, and tweaks to the generator.  In the process, a couple of clarifications were made -- minor, to do with things that have changed over time.

### CODE REVIEW

**Comment by swcurran** [comment]
Lots of files changed, but only a few things to look at:

* Index and mkdocs have the same set of changes as the last change.  Not entirely sure why they are being redone.
  * If the index/mkdocs are right, the statuses are right as the those files are generated.
* Proper "mailto:" added to all the Authors lines. Trust me 🙂
* Clean ups of naked URLs and other things that generated mkdocs warnings/errors.

So really -- just make sure that the index is right and all good.

And one more note -- RFC 0685 used to have as it's title line "RFC 0212", so that is fixed (someone forgot to change the RFC number in the title), and PR 0799 has moved from top to bottom of proposed - back in alpha order.

File: features/0193-coin-flip/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Aries RFC 0193: Coin Flip Protocol 1.0 
 
-- Authors: [Daniel Hardman](daniel.hardman@gmail.com), [Patrick Stürmlinger](patrick@mindf.org)
-- Status: [PROPOSED](/README.md#proposed)
-- Since: 2019-08-19
-- Status Note: minor update in late 2020 
+- Authors: [Daniel Hardman](mailto:daniel.hardman@gmail.com), [Patrick Stürmlinger](pmailto:atrick@mindf.org)
```
Comment (JamesKEbert) [inline features/0193-coin-flip/README.md:3]:
```suggestion
- Authors: [Daniel Hardman](mailto:daniel.hardman@gmail.com), [Patrick Stürmlinger](mailto:patrick@mindf.org)
```

File: concepts/0250-rich-schemas/README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # RFC 0250: Rich Schema Objects
 - Author: [Ken Ebert](ken@sovrin.org), [Brent Zundel](brent.zundel@evernym.com)
```
Comment (JamesKEbert) [inline concepts/0250-rich-schemas/README.md:2]: Missing `mailto:`

File: concepts/0217-linkable-message-paths/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Aries RFC 0217: Linkable Message Paths
 - Authors: [Daniel Hardman](daniel.hardman@gmail.com)
```
Comment (JamesKEbert) [inline concepts/0217-linkable-message-paths/README.md:2]: Missing `mailto:`

File: concepts/0207-credential-fraud-threat-model/README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # 0207: Credential Fraud Threat Model
 - Authors: [Daniel Hardman](daniel.hardman@gmail.com)
```
Comment (JamesKEbert) [inline concepts/0207-credential-fraud-threat-model/README.md:2]: Missing `mailto:`

File: concepts/0094-cross-domain-messaging/README.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 # Aries RFC 0094: Cross-Domain Messaging
 
 - Authors: [Stephen Curran](swcurran@gmail.com)
```
Comment (JamesKEbert) [inline concepts/0094-cross-domain-messaging/README.md:3]: Missing `mailto:`

File: concepts/0051-dkms/README.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
 # Aries RFC 0051: Decentralized Key Management
 
 - Authors: [Drummond Reed](drummond@connect.me) et al.
```
Comment (JamesKEbert) [inline concepts/0051-dkms/README.md:3]: Missing `mailto:`

File: concepts/0029-message-trust-contexts/README.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
 # Aries RFC 0029: Message Trust Contexts
 
 - Authors: [Daniel Hardman](daniel.hardman@gmail.com)
```
Comment (JamesKEbert) [inline concepts/0029-message-trust-contexts/README.md:3]: Missing `mailto:`

File: concepts/0017-attachments/README.md [hunk 8]
```diff
@@ -1,8 +1,8 @@
 # Aries RFC 0017: Attachments
 
 - Authors: [Daniel Hardman](daniel.hardman@gmail.com), Sam Curren, Andrew Whitehead
```
Comment (JamesKEbert) [inline concepts/0017-attachments/README.md:3]: Missing `mailto:`

**Review by JamesKEbert — CHANGES_REQUESTED** [review]
Looks good. Just some minor catches

**Review by JamesKEbert — APPROVED** [review]
LGTM

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**